src/com/github/argajuvi/menus/ProductMenu.java

```diff
@@ -0,0 +1,215 @@
+                    "Quantity", "Total Price"
+            );
+            System.out.print(line);
+
+            int totalOfTotalPrice = 0;
+            for (Order order : cart) {
+                count++;
+
+                Product product = order.getProduct();
+                totalOfTotalPrice += order.getTotalPrice();
+
+                System.out.printf(
+                        rowFormat,
+                        count + "",
+                        product.getName(),
+                        Utils.formatPriceIDR(product.getPrice()), product.getType().getName(),
+                        order.getQuantity() + "",
+                        Utils.formatPriceIDR(order.getTotalPrice())
+                );
+            }
+
+            System.out.print(line);
+            System.out.printf("| %-46s | %-69s |\n", "Total Price", Utils.formatPriceIDR(totalOfTotalPrice));
+            System.out.print(line);
+        }
+    }
+
+    public static void showCartProducts() {
+        Utils.clearScreen();
+
+        List<Order> cart = Main.currentUser.getCart();
+        showCartView();
+
+        if (cart.isEmpty()) {
+            Utils.waitForEnter();
+            return;
+        }
+
+        int choice;
+        System.out.println();
+
+        while (true) {
+            choice = Utils.scanAbsoluteInt("Remove order ['0' to go back]: ");
+
+            if (choice == 0) {
+                return;
+            }
+            if (choice < 1 || choice > cart.size()) {
+                System.out.println("Cannot find order");
+                continue;
+            }
+
+            break;
+        }
+
+        cart.remove(choice - 1);
+        System.out.println("Successfully removed order from the cart");
+
+        Utils.waitForEnter();
+    }
+
+    public static void showCheckout() {
+        Utils.clearScreen();
+
+        List<Order> cart = Main.currentUser.getCart();
+        showCartView();
+
+        if (cart.isEmpty()) {
+            Utils.waitForEnter();
+            return;
+        }
+
+        boolean isConfirmed = Utils.scanAbsoluteConfirmation("Do you want to purchase those products? [y/n]: ");
+        if (!isConfirmed) {
+            return;
+        }
+
+        List<Order> orderList = new ArrayList<>(cart);
+        int totalOfTotalPrice = 0;
+
+        for (Order order : orderList) {
+            totalOfTotalPrice += order.getTotalPrice();
+        }
+
+        Receipt receipt = new Receipt(orderList, LocalDate.now(), totalOfTotalPrice);
+        Main.currentUser.getReceiptList().add(receipt);
+        cart.clear();
+
+        System.out.println("Successfully purchase products!");
+        Utils.waitForEnter();
+    }
+
+    /**
+     * Shows the list of products in a table view
+     *
+     * @param filterType the type of product should be shown on the view,
+     *                   insert {@code null} to not use any filters
+     */
+    public static void showProductsView(@Nullable Product.Type filterType) {
```

Comment:
I have to say, this isn't the best way to implement it but I don't really mind it.

Menurut gw sih ini bisa buat 1 class misalnya `Views` di package `utils` trus masukin itu semua function yg berhubungan sama _views_, ato kalo mau versi malesnya taro di class `Main`